### ISSUE TITLE
Destruction particle bug on  ̷Q̷u̷i̷l̷t̷ Continuity

### ISSUE THREAD

**Issue by kastrom3** [FirEmerald/DynamicVerticalSlabsAndStairs#71]
When vertical slabs, steps, carpets and plate are destroyed, missing texture particles appear instead of normal particles. Moreover, if you break the vertical double slab, then everything is fine. There are many other mods in the build, but it seems to be a problem with the Quilted Fabric API.
Here is the error from the console:
```
[16:49:50] [Render thread/INFO]: 
java.lang.Exception: null
	at com.firemerald.additionalplacements.client.models.BakedPlacementBlockModel.method_4711(BakedPlacementBlockModel.java:66) ~[additionalplacements-1.8.0.i0:0/:?]
	at net.fabricmc.fabric.api.renderer.v1.model.ForwardingBakedModel.method_4711(ForwardingBakedModel.java:82) ~[quilted-fabric-renderer-api-v1-7.5.00.91.0-1.20.1.i0:0/:?]
	at net.fabricmc.fabric.api.renderer.v1.model.ForwardingBakedModel.method_4711(ForwardingBakedModel.java:82) ~[quilted-fabric-renderer-api-v1-7.5.00.91.0-1.20.1.i0:0/:?]
	at net.fabricmc.fabric.api.renderer.v1.model.ForwardingBakedModel.method_4711(ForwardingBakedModel.java:82) ~[quilted-fabric-renderer-api-v1-7.5.00.91.0-1.20.1.i0:0/:?]
	at net.minecraft.class_773.method_3339(class_773.java:23) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_727.<init>(class_727.java:25) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_702.method_34020(class_702.java:498) ~[minecraft-1.20.1.i0:0/:?]
	at me.jellysquid.mods.lithium.common.shapes.VoxelShapeSimpleCube.method_1089(VoxelShapeSimpleCube.java:201) ~[lithium-0.11.2.i0:0/:?]
	at net.minecraft.class_702.method_3046(class_702.java:478) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_638.method_31595(class_638.java:1030) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_761.method_8567(class_761.java:2922) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_638.method_8444(class_638.java:584) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_2248.method_33614(class_2248.java:481) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_2248.method_9576(class_2248.java:486) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_636.method_2899(class_636.java:127) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_636.method_41936(class_636.java:157) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_636.method_41931(class_636.java:278) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_636.method_2910(class_636.java:156) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_310.method_1536(class_310.java:1736) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_310.method_1508(class_310.java:2049) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_310.method_1574(class_310.java:1883) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_310.method_1523(class_310.java:1181) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.class_310.method_1514(class_310.java:802) ~[minecraft-1.20.1.i0:0/:?]
	at net.minecraft.client.main.Main.main(Main.java:250) ~[minecraft-1.20.1-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:551) ~[quilt-loader-0.23.0.jar:?]
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:84) ~[quilt-loader-0.23.0.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:28) ~[quilt-loader-0.23.0.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:243) ~[NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:278) ~[NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) ~[NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) ~[NewLaunch.jar:?]
```

https://github.com/FirEmerald/AdditionalPlacements/assets/69670223/de835281-d53a-4415-9c2e-12167e16bb7a

**Comment by kastrom3** [comment]
This error appears due to the Continuity mod in combination with mods and, most likely, resource packs using the Optifine folder.
During a test by elimination, I found that if I disable Continuity, everything works fine. Also, just in case, I tested only with these 2 mods and those necessary for their work, and everything also worked fine. According to this, using the same method, I found 3 other mods in combination with which Continuity breaks Additional Placements particles.
These were:
From The Fog in the Optifine folder there is only emissive.properties
Supplementaries is similar to the previous one
My mod in which I added a connection to several blocks from the mod in the Optifine folder, and a random texture to the sheep.
